### ISSUE TITLE
Make topic link auto-clickable

### ISSUE THREAD

**Issue by vikaskedia** [RocketChat/Rocket.Chat#6595]
the channel topic if it has a link should be clickable

![image](https://cloud.githubusercontent.com/assets/6703876/24673367/eb4deb7a-192c-11e7-9505-ba5d4c188f3c.png)

it would help me if I could click on the URL in the channel topic.

**Comment by geekgonecrazy** [comment]
If you need it immediately clickable you can do `[url](url)` and it'll become clickable :)

**Comment by vikaskedia** [comment]
Tx :)

**Comment by Nepoxx** [comment]
Can we close this?

**Comment by geekgonecrazy** [comment]
I think still applies.  Edited topic to reflect the request

**Comment by knrt10** [comment]
On it, will try to fix this

**Comment by gelpiu-developers** [comment]
There's another issue with links in topic... When you set a non http / https link it doesn't work. For instance, using a custom protocol for deep-linking.

**Comment by thw0rted** [comment]
I agree that links should be parsed automatically, but would also like to see clickable channel references in a topic (like in a regular message).  For example, you should be able to click the channel name if the topic is e.g. `See also #otherchannel`.

Should I open a separate issue to track that?

**Comment by aurisnoctis** [comment]
Clickable links in channel topics for references like `#mychannel` would make life easier on Rocket.Chat. 🙌 

See https://github.com/RocketChat/Rocket.Chat/issues/27491 for problems with markdown links in channel topics, specifically https://github.com/RocketChat/Rocket.Chat/issues/27491#issuecomment-1842847119.

**Comment by MartinSchoeler** [comment]
Should be working with the latest version of rocket.chat

<img width="1596" height="221" alt="Image" src="https://github.com/user-attachments/assets/852d044e-368a-4400-9972-9bf80261f465" />
